src/components/Slider/Slider.module.scss

```diff
@@ -13,7 +15,7 @@
   display: flex;
   align-items: center;
   width: 18.75rem;
-  background-color: #37273f;
+  background-color: #{$purple-light-4};
```

Comment:
Awesome thanks Tim!

I'm fairly certain you could do `background-color: $purple-light-4` rather than `#{$purple-light-4}`. Not a big deal, as long as either work 🙂

Reply:
Done